src/artworks/0_artworks-list/artworksList.tsx

```diff
@@ -1,39 +1,55 @@
-import {ReactNode} from "react";
-import Template from "../0_template/Template.tsx"
-import {EArtworkTags} from "./artworkTags.ts";
+import { ReactNode } from "react";
+import Template from "../0_template/Template.tsx";
+import { EArtworkTags } from "./artworkTags.ts";
+import AnalogClock from "../AnalogClock/AnalogClock.tsx";
 
 export interface ILink {
-    name: string;
-    href: string;
+  name: string;
+  href: string;
 }
 
 export interface IArtworkItem {
-    component: ReactNode;
-    titel: string;
-    description: string;
-    gitHubName: string;
-    tags?: EArtworkTags[] | string[]
-    backgroundColor?: `#${string}`;     // No transparent backgrounds
-    full?: boolean;                     // Only use this if you really have no other but to claim more space.
-    oneLink?: ILink;
-    editedAt?: Date;
+  component: ReactNode;
+  titel: string;
+  description: string;
+  gitHubName: string;
+  tags?: EArtworkTags[] | string[];
+  backgroundColor?: `#${string}`; // No transparent backgrounds
+  full?: boolean; // Only use this if you really have no other but to claim more space.
+  oneLink?: ILink;
+  editedAt?: Date;
 }
 
 const artworksList: IArtworkItem[] = [
-    {
-        component: <Template />,
-        titel: "circle",
-        description: "round circle",
-        gitHubName: "FabDonRixos",
-        tags: [EArtworkTags.SIMPLE],
-        backgroundColor: "#000",
-        full: true,
-        oneLink: {
-            name: "fabian.li",
-            href: "https://fabian.li"
-        },
-        editedAt: new Date("2024-10-26")
+  {
+    component: <Template />,
+    titel: "circle",
+    description: "round circle",
+    gitHubName: "FabDonRixos",
+    tags: [EArtworkTags.SIMPLE],
+    backgroundColor: "#000",
+    full: true,
+    oneLink: {
+      name: "fabian.li",
+      href: "https://fabian.li",
     },
-]
+    editedAt: new Date("2024-10-26"),
+  },
+  {
+    component: <AnalogClock />,
+    titel: "Analog Clock",
+    description:
+      "A simple analog clock with hour, minute, and second hands animated in CSS.",
+    gitHubName: "duyb2111918",
+    tags: [EArtworkTags.COMPLEX, "TIME"],
```

Comment:
I would add the tag animation and please only use tags inside EArtworkTags because it won't be possible to find it with tags if you make your own, also if you want to be able to find you component if you type "time" its better to add time to the title or the description.